build.sbt

```diff
@@ -121,3 +122,26 @@ lazy val driver = (project in file("driver"))
       ++ Dependencies.TestDriver.dependencies
       :+ ("org.scala-lang" % "scala-reflect" % scalaVersion.value)
   )
+
+/** The following two projects define an extra artifact published alongside main 'spark-cassandra-connector'.
+  * It's a shaded version of the main artifact. It contains all of the dependent classes, some of them are shaded. */
+lazy val assembledJar = project
+  .enablePlugins(AssemblyPlugin)
+  .settings(assemblySettings)
+  .settings(
+    publish / skip := true,
+    assembly / assemblyOption := (assemblyOption in assembly).value.copy(includeScala = false),
+    assembly / assemblyShadeRules := {
+      Seq(
+        ShadeRule.rename("com.typesafe.config.**" -> s"shade.com.datastax.spark.connector.com.typesafe.config.@1").inAll
```

Comment:
we also need to shade the `com.typesafe.scalalogging` that is used by `com.typesafe.config`

Reply:
scala-logging was our dependency, I removed it in one of the previous commits

Reply:
ah, ok... I messed it up :-)